Automattic/jetpack
# autoloader, composer-plugin: Sort data in generated files

#### Changes proposed in this Pull Request:

We don't get the packages to process in a consistent order, and the order of the data in the associative arrays in composer-plugin's `i18n-map.php` and the autoloader's `jetpack_autoloader_classmap.php` doesn't matter. So let's sort those arrays so diffs are cleaner.

Note the data for the autoloader files (`jetpack_autoloader_filemap.php` and `jetpack_autoloader_psr4.php`) is already sorted when it's generated, so we don't have to do anything for those.

#### Other information:


#### Jetpack product discussion


None really. I noticed spurious diffs when checking a recent release, and decided to fix them.

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Run `composer install` in various plugins and see that the data in the mentioned files is now sorted.
  * If you're curious about the files we don't change here: `jetpack_autoloader_filemap.php` should be in order by dependency, and the namespace name array in `jetpack_autoloader_psr4.php` should be in reverse order (because that's how Composer does it).
* Everything should continue to auto-load as it did before.
